cloud/amazon/sns_topic.py

```diff
@@ -97,35 +97,34 @@
 """
 
 RETURN = '''
-topic_created:
-    description: Whether the topic was newly created
+changed:
```

Comment:
changed is a common return and must not be added in returns doc. Please remove it.